Move firewall doc under "Application" section (#534)
NethServer/docs: administrator-manual/en/vpn.rst

2. :index:`net2net` or :index:`tunnel`: connect two remote networks

Roadwarrior OpenVPN, IPSec tunnel and OpenVPN tunnels are part of the VPN application in the new Server Manager.

> By default, network traffic between VPNs is blocked by firewall. To allow such traffic enable the :guilabel:`Allowed` field under
> the :guilabel:`Traffic between OpenVPN roadwarrior, OpenVPN tunnels and IPSec tunnels` inside the :guilabel:`Settings` 
> page of :ref:`firewall-section` application. 

Reviewer: ```suggestion
By default, the network traffic between VPNs is blocked by the firewall. 
To allow such traffic go to the :ref:`firewall-section` application and enable the :guilabel:`Allowed` field under
:guilabel:`Settings > Traffic between OpenVPN roadwarrior, OpenVPN tunnels and IPSec tunnels`.
```
